From b4956012e062dd509274676300ced8792e2d88a6 Mon Sep 17 00:00:00 2001
From: Yu Kang Ku <yu.kang.ku@intel.com>
Date: Thu, 8 Oct 2020 09:41:24 -0700
Subject: [PATCH 1/2] Disable IPC usage

This allows iHD to be used within the Chrome OS sandbox.

Change-Id: I7c9d438ab7072bbeb35b295a624b71e310ade6f7
Signed-off-by: Yu Kang Ku <yu.kang.ku@intel.com>
---
 media_driver/linux/common/os/mos_context_specific.cpp | 2 ++
 media_driver/linux/common/os/mos_os_specific.c        | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/media_driver/linux/common/os/mos_context_specific.cpp b/media_driver/linux/common/os/mos_context_specific.cpp
index ebdf1343..e575e6cf 100644
--- a/media_driver/linux/common/os/mos_context_specific.cpp
+++ b/media_driver/linux/common/os/mos_context_specific.cpp
@@ -32,6 +32,8 @@
 #include "hwinfo_linux.h"
 #include <stdlib.h>
 
+#define ANDROID 1
+
 #ifndef ANDROID
 #include <sys/ipc.h>
 #include <sys/shm.h>
diff --git a/media_driver/linux/common/os/mos_os_specific.c b/media_driver/linux/common/os/mos_os_specific.c
index fb60a1fc..9687a8b0 100644
--- a/media_driver/linux/common/os/mos_os_specific.c
+++ b/media_driver/linux/common/os/mos_os_specific.c
@@ -50,6 +50,8 @@
 #endif // MOS_MEDIASOLO_SUPPORTED
 #include "mos_solo_generic.h"
 
+#define ANDROID 1
+
 #ifndef ANDROID
 #include <sys/ipc.h>
 #include <sys/shm.h>
-- 
2.17.1

